contents/blog/the-posthog-array-1-10-0.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2020-06-25
+title: Array 1.10.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+This is our most exciting release to date. Here we go: 🎉
```

Comment:
I remember from somewhere that it's not "our style" to use emojis! What happened! 😲